lib/wait-for-ok.js

```diff
@@ -5,23 +5,31 @@ module.exports = function (url) {
 		let timeout;
 		let checker;
 		function checkGtg () {
-			console.log(`polling: ${url}`); // eslint-disable-line no-console
+			console.log(`⏳ polling: ${url}`); // eslint-disable-line no-console
 			fetch(url, { timeout: 2000, follow: 0 })
 				.then(function (response) {
 					if (response.ok) {
-						console.log(`poll ${url} ok`); // eslint-disable-line no-console
+						console.log(`⌛️ ${url} ok!`); // eslint-disable-line no-console
 						clearTimeout(timeout);
 						clearInterval(checker);
 						resolve();
 					} else {
-						console.log(`poll ${url} not ok`); // eslint-disable-line no-console
+						console.log(`⌛️ ${url} not ok`); // eslint-disable-line no-console
```

Comment:
...and a ❌ for when they're not.

I figure these are easy to scan for in output to determine success or failure and similar to the output that test runners like mocha and n-test provide.

:thinking: